en/peripherals/uavcan_escs.md

```diff
@@ -1,20 +1,20 @@
 # UAVCAN ESCs (Motor Controllers)
 
-PX4 supports the [UAVCAN](https://uavcan.org/) bus for connecting peripherals, including ESCs, GPS modules, various types of sensors, etc.
+PX4 supports the [UAVCAN](https://uavcan.org/) bus for connecting peripherals, including ESCs, GPS modules, various types of sensors, etc. However PX4 currently does not support UAVCAN Servos.
 
-UAVCAN ESCs have a number of advantages over [PWM ESCs and Servos](../peripherals/pwm_escs_and_servo.md):
+UAVCAN ESCs have a number of advantages over [PWM ESCs](../peripherals/pwm_escs_and_servo.md):
 - UAVCAN has been specifically designed to deliver robust and reliable connectivity over relatively large distances.
   It enables safe use of ESCs on bigger vehicles and communication redundancy.
 - The bus is bi-directional, enabling health monitoring and diagnostics.
 - Wiring is less complicated as you can have a single bus for connecting all your ESCs and other UAVCAN peripherals.
-- Setup is easier as you configure ESC numbering by manually spinning each motor.
+- Setup is easier as you configure ESC numbering by manually spinning each motor (in most cases).
```

Comment:
What does this add and is it true - ie you spin the motors, and then you can define the numbering. ISn't that easier?

Reply:
In some cases the numbering is not easier than with PWM ESCs and it doesn't work by manually spinning each motor. I'm talking about VESC where you have to manually set the index of each VESC, both for UAVCAN ID and for VESC ID. I'd argue that this setup is even more cumbersome than PWM ESCs, hence the small note.

Reply:
Thanks!